Mindee Invoice.ipynb

```diff
@@ -0,0 +1,417 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "#load image - get height and width - show image\n",
+    "imagepath = r'invoice1.png'\n",
+    "image = cv2.imread(imagepath)\n",
+    "#fix colour space\n",
+    "image = cv2.cvtColor(image, cv2.COLOR_BGR2RGB)\n",
+    "height, width, channels = image.shape\n",
+    "print (height, width)\n",
+    "plt.imshow(image)\n",
+    "plt.show()"
```

Comment:
Not sure whether you want to keep the indices on the axis, but if not, you can add `plt.axis("off")` 1 line before `plt.show()`

Reply:
IDK if we need to go this deep. I think each python dev can update as they desire.

Reply:
Sure, it was only a suggestion :+1: